bom/pom.xml

```diff
@@ -707,6 +707,27 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
+              <groupId>io.rest-assured</groupId>
+              <artifactId>json-schema-validator</artifactId>
+              <version>${rest-assured.version}</version>
```

Comment:
Sorry, I missed that the first time but the indentation is incorrect: it should be indented with 4 spaces.

Could you fix that?

Then I think it's good to go!